public/Disable-DbaReplDistributor.ps1

```diff
@@ -0,0 +1,102 @@
+function Disable-DbaReplDistributor {
+    <#
+    .SYNOPSIS
+        Disables replication distribution for the target SQL instances.
+
+    .DESCRIPTION
+        Disables replication distribution for the target SQL instances.
```

Comment:
```suggestion
        Uninstalls the distributor database on the specific SQL Server instance.
        This command is equivalent to running sp_dropdistributor stored procedure in SQL Server.
```

If this is true as what I linked above and the bottom of that doc page references this as being the equivalent I'd still say this needs to use Uninstall verb.